tools/ldb_cmd.cc

```diff
@@ -3770,6 +3807,48 @@ void DBQuerierCommand::DoCommand() {
           oss << "get " << key << " error: " << s.ToString();
         }
       }
+    } else if (cmd == COUNT_CMD) {
+      std::string start_key;
+      std::string end_key;
+      bool bad_option = false;
```

Comment:
We may need some general options sanity check logic, like if `get` has `--from` and `--to`, it should error out too? I'm working on adding `--read_timestamp=<uint64_ts>` to querier that will piggyback the parsing logic you added. I can add these sanity checks there too.